tortoise_interpreter.py

```diff
@@ -20,41 +19,59 @@ def sanitize_program(program):
 
 
 def navigate_command_functions(COMMANDS, program):
-    # Comment
-    return [f'{i}. {COMMANDS[command[0]](command)}' for i, command in enumerate(program, 1)]
+    command_to_parse = []
+    pen_state = ''
```

Comment:
Should this default to empty? Is there a sensible default you could use? One that's implied by the ticket? :innocent:

Reply:
done